cypress/e2e/utils/env.ts

```diff
@@ -32,8 +32,8 @@ export const loadPlayer = (pluginConf = {}, playbackConf: Record<string, any> =
               download: {
                 downloads: 'Downloads',
                 download: 'Download',
-                select_quality_label: 'Select quality',
-                hide_label: 'Hide',
+                select_quality_label: 'More flavors',
+                hide_label: 'Less flavors',
```

Comment:
is this the text in the spec ? i think "fewer flavors" would be the right term

Reply:
Download plugin should use 
```
                select_quality_label: 'Select quality',
                hide_label: 'Hide',
```